Epam.TestAutomation.Tests/CareersPageTests.cs

```diff
@@ -37,20 +40,130 @@ public void HoverOverCareerMenuTest()
     }
 
     [Test]
-    public void DDTKeyWordPhrases()
+    [TestCaseSource(nameof(GetSearchTestData1))]
+    public void DDTKeyWordPhrases(SearchModel searchModel)
```

Comment:
Change the name of the test. And add Test to the end of the method's name for all classes.